### PR TITLE
OSASINFRA-3560: OpenStack: kubelet-nodename service for IPv6

### PR DESCRIPTION
**Pull request by MaysaMacedo**
**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: templates/common/openstack/files/usr-local-bin-openstack-kubelet-nodename.yaml [hunk 1]
```diff
@@ -30,8 +30,13 @@ contents:
     # For compatibility with the OpenStack in-tree provider
     # Set node name to be instance name instead of the default FQDN hostname
     while true; do
-        # https://docs.openstack.org/nova/victoria/user/metadata.html#metadata-openstack-format
-        hostname=$(curl -s http://169.254.169.254/openstack/2012-08-10/meta_data.json | jq -re .name)
+        if [ "$SINGLE_STACK_IPV6" = true ]
+        then
+            # https://docs.openstack.org/nova/victoria/user/metadata.html#metadata-openstack-format
+            hostname=$(curl -s http://169.254.169.254/openstack/2012-08-10/meta_data.json | jq -re .name)
```
[the remark this answers is not in the document]
Comment (MaysaMacedo) [reply templates/common/openstack/files/usr-local-bin-openstack-kubelet-nodename.yaml:38]: ops, yes!

File: templates/common/openstack/files/usr-local-bin-openstack-kubelet-nodename.yaml [hunk 2]
```diff
@@ -30,8 +30,13 @@ contents:
     # For compatibility with the OpenStack in-tree provider
     # Set node name to be instance name instead of the default FQDN hostname
     while true; do
-        # https://docs.openstack.org/nova/victoria/user/metadata.html#metadata-openstack-format
-        hostname=$(curl -s http://169.254.169.254/openstack/2012-08-10/meta_data.json | jq -re .name)
+        if [ "$SINGLE_STACK_IPV6" = true ]
+        then
+            hostname=$(curl http://[fe80::a9fe:a9fe%25br-ex]:80/openstack/2012-08-10/meta_data.json | jq -re .name)
```
Comment (EmilienM) [inline templates/common/openstack/files/usr-local-bin-openstack-kubelet-nodename.yaml:35]: I think this is fine since at this point br-ex should exist.

**Pull request merged**